File: app/src/main/java/com/freetime/appstore/german/main/DE_MainActivity.java
package com.freetime.appstore.german.main;

import static com.freetime.appstore.R.id;
import static com.freetime.appstore.R.layout;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

import com.freetime.appstore.R;
import com.freetime.appstore.english.apps.freetime.Dono_Activity;
import com.freetime.appstore.english.apps.freetime.FAS_Activity;
import com.freetime.appstore.english.apps.freetime.GW_Activity;
import com.freetime.appstore.english.apps.freetime.KPDHJ_Activity;
import com.freetime.appstore.english.apps.freetime.RadioPlayerJavaActivity;
import com.freetime.appstore.english.contributions.GH_Activity;
import com.freetime.appstore.english.games.freetime.CatClickerActivity;
import com.freetime.appstore.english.games.freetime.CryptoClickerActivity;
import com.freetime.appstore.english.games.freetime.OS_Activity;
import com.freetime.appstore.english.games.freetime.PLC2Activity;
import com.freetime.appstore.english.games.freetime.PLC_Activity;
import com.freetime.appstore.english.games.freetime.PlatformerActivity;
import com.freetime.appstore.english.main.MainActivity;
import com.freetime.appstore.german.apps.freetime.SSMPCJ_Activity;


public class DE_MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(layout.activity_de_main);

        Button gwInfobtn = findViewById(id.gwInfobtn);
        gwInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, GW_Activity.class);
            startActivity(intent);
        });

        Button donoInfobtn = findViewById(id.donoInfobtn);
        donoInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, Dono_Activity.class);
            startActivity(intent);
        });

        Button fasInfobtn = findViewById(id.fasInfobtn);
        fasInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, FAS_Activity.class);
            startActivity(intent);
        });

        Button catInfobtn = findViewById(id.catInfobtn);
        catInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, CatClickerActivity.class);
            startActivity(intent);
        });

        Button PLCInfobtn = findViewById(id.PLCInfobtn);
        PLCInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, PLC_Activity.class);
            startActivity(intent);
        });

        Button plc2Infobtn = findViewById(id.plc2Infobtn);
        plc2Infobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, PLC2Activity.class);
            startActivity(intent);
        });

        Button platInfobtn = findViewById(id.platInfobtn);
        platInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, PlatformerActivity.class);
            startActivity(intent);
        });

        Button osInfobtn = findViewById(id.osInfobtn);
        osInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, OS_Activity.class);
            startActivity(intent);
        });

        Button cryptoInfobtn = findViewById(id.cryptoInfobtn);
        cryptoInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, CryptoClickerActivity.class);
            startActivity(intent);
        });

        Button RPJInfobtn = findViewById(id.RPJInfobtn);
        RPJInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, RadioPlayerJavaActivity.class);
            startActivity(intent);
        });

        Button KPDHJInfobtn = findViewById(R.id.KPDHJInfobtn);
        KPDHJInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, KPDHJ_Activity.class);
            startActivity(intent);
        });

        Button SSMPCJInfobtn = findViewById(R.id.SSMPCJInfobtn);
        SSMPCJInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, SSMPCJ_Activity.class);
            startActivity(intent);
        });

        Button ghInfobtn = findViewById(id.ghInfobtn);
        ghInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, GH_Activity.class);
            startActivity(intent);
        });

        Button enInfobtn = findViewById(id.enInfobtn);
        enInfobtn.setOnClickListener(v -> {
            Intent intent = new Intent(DE_MainActivity.this, MainActivity.class);
            startActivity(intent);
        });
    }
}

File: app/src/main/java/com/freetime/appstore/german/main/DE_MainActivity.kt
package com.freetime.appstore.german.main

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import com.freetime.appstore.R
import com.freetime.appstore.english.apps.freetime.*
import com.freetime.appstore.english.contributions.GH_Activity
import com.freetime.appstore.english.main.MainActivity
import com.freetime.appstore.german.apps.freetime.SSMPCJ_Activity

class DE_MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_de_main)

        val gwInfobtn: Button = findViewById(R.id.gwInfobtn)
        gwInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, GW_Activity::class.java))
        }

        val donoInfobtn: Button = findViewById(R.id.donoInfobtn)
        donoInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, Dono_Activity::class.java))
        }

        val fasInfobtn: Button = findViewById(R.id.fasInfobtn)
        fasInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, FAS_Activity::class.java))
        }

        val catInfobtn: Button = findViewById(R.id.catInfobtn)
        catInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, CatClickerActivity::class.java))
        }

        val PLCInfobtn: Button = findViewById(R.id.PLCInfobtn)
        PLCInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, PLC_Activity::class.java))
        }

        val plc2Infobtn: Button = findViewById(R.id.plc2Infobtn)
        plc2Infobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, PLC2Activity::class.java))
        }

        val platInfobtn: Button = findViewById(R.id.platInfobtn)
        platInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, PlatformerActivity::class.java))
        }

        val osInfobtn: Button = findViewById(R.id.osInfobtn)
        osInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, OS_Activity::class.java))
        }

        val cryptoInfobtn: Button = findViewById(R.id.cryptoInfobtn)
        cryptoInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, CryptoClickerActivity::class.java))
        }

        val RPJInfobtn: Button = findViewById(R.id.RPJInfobtn)
        RPJInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, RadioPlayerJavaActivity::class.java))
        }

        val KPDHJInfobtn: Button = findViewById(R.id.KPDHJInfobtn)
        KPDHJInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, KPDHJ_Activity::class.java))
        }

        val SSMPCJInfobtn: Button = findViewById(R.id.SSMPCJInfobtn)
        SSMPCJInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, SSMPCJ_Activity::class.java))
        }

        val ghInfobtn: Button = findViewById(R.id.ghInfobtn)
        ghInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, GH_Activity::class.java))
        }

        val enInfobtn: Button = findViewById(R.id.enInfobtn)
        enInfobtn.setOnClickListener {
            startActivity(Intent(this@DE_MainActivity, MainActivity::class.java))
        }
    }
}
